forms-flow-idm/keycloak/Dockerfile

```diff
@@ -1,5 +1,9 @@
-FROM jboss/keycloak
+FROM formsflow/forms-flow-idm-theme
```

Comment:
change image version to formsflow/forms-flow-idm-theme:v7.0

Reply:
@amal-k-soman-aot  please check